frontend/src/lib/modals/neurons/DisburseSnsNeuronModal.svelte

```diff
@@ -96,15 +96,15 @@
 
     stopBusy("disburse-sns-neuron");
 
+    dispatcher("nnsClose");
+
     if (success) {
       toastsSuccess({
         labelKey: "neuron_detail.disburse_success",
       });
 
       await goto($neuronsPathStore, { replaceState: true });
```

Comment:
I would also remove the await part, to not block the modal close. Because when navigation is done, the modal should be destroyed, not starting the close animation. What do you think about this change?
```suggestion
      goto($neuronsPathStore, { replaceState: true });
```

Reply:
Ah good to know, thanks!